dev_mode/webpack.prod.minimize.config.js

```diff
@@ -21,7 +21,6 @@ config[0] = merge(config[0], {
     minimizer: [
       new TerserPlugin({
         parallel: true,
-        sourceMap: true,
```

Comment:
The plugin respects `devtool` by default now: https://github.com/webpack-contrib/terser-webpack-plugin/blob/master/CHANGELOG.md#-breaking-changes